CHANGELOG.md

```diff
@@ -0,0 +1,687 @@
+# Change Log
+
+## [Unreleased](https://github.com/confluentinc/ksql/tree/HEAD)
```

Comment:
When does the change log get generated? Maybe leave the unreleased stuff out until we establish some cadence for generating it.

Reply:
Agreed. This is my first pass at using the tool and it is definitely possible to skip the unreleased stuff.